recipes/jags/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "JAGS" %}
+{% set version = "4.3.0" %}
+{% set sha256 = "8ac5dd57982bfd7d5f0ee384499d62f3e0bb35b5f1660feb368545f1186371fc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://sourceforge.mirrorservice.org/m/mc/mcmc-jags/JAGS/4.x/Source/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - gcc
```

Comment:
Do you actually explicitly need the `gcc` package on Linux? Seems unlikely. If it doesn't work with the system clang on Mac, you should probably just do <code>- gcc &nbsp;# [osx]</code> (and similar for `libgcc` in `run`).